float32 setpoint
time stamp